tasks/docker.yml

```diff
@@ -0,0 +1,37 @@
+---
+
+- name: HAProxy - run a Docker container
+  community.general.docker_container:
```

Comment:
Using the FQCN will break the compatibility with previous Ansible version.

Reply:
@goldyfruit, that's true.
What's the minimum version of Ansible you want to support? 
Because using old style for `docker_container` will produce a warning under 2.10.

As a transition though, can add a check via `ansible_version` and use the required module, e.g. https://stackoverflow.com/questions/34809845/check-ansible-version-from-inside-of-a-playbook

Reply:
> @goldyfruit, that's true.
> What's the minimum version of Ansible you want to support?
2.8 should be reasonable.
> As a transition though, can add a check via `ansible_version` and use the required module, e.g. https://stackoverflow.com/questions/34809845/check-ansible-version-from-inside-of-a-playbook
This could be a solution until 2.10 become the new "normal".

Reply:
Done.